Alc/backends/null.cpp

```diff
@@ -96,7 +96,7 @@ int NullBackend::mixerProc()
          * and current time from growing too large, while maintaining the
          * correct number of samples to render.
          */
-        if(done >= mDevice->Frequency)
+        if(done >= mDevice->Frequency && mDevice->Frequency != 0)
```

Comment:
It's impossible for `Frequency` to be 0 (it's initialized to 44100 by default, and the minimum allowed is 8000). If it's 0, something else is very wrong and many other problems will occur. Perhaps an `ASSUME(device-Frequency > 0);` statement can silence clang-analyzer?